### DISCUSSION TITLE
Autocomplete group names

### DISCUSSION THREAD

**Discussion by vanillajonathan** [NickvisionApps/Denaro#113] [Ideas]
To make it easier to use and know what to use groups for, Money could offer autocomplete suggestions.

Maybe something like...
- Bills
- Clothes
- Food
- Entertainment
- Subscriptions

**Comment by nlogozzo** [comment]
@fsobolev what do you think?

**Reply by fsobolev** [reply]
I don't think it's hard to understand what to use groups for. And they obviously can be used for basically anything. A user might want to track how much money they spend on coffee, or how much they get from patreon, or to track what they buy and sell on an auction in an online game. Suggestions will be insufficient anyway. Even the presented list only contain possible groups for expenses, not incomes. I think we should not guess what a user uses the app for, and thus do not suggest anything.

**Reply by nlogozzo** [reply]
I was thinking the same thing.
